docs/src/main/paradox/slick.md

```diff
@@ -0,0 +1,139 @@
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-slick" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.lightbend.akka</groupId>
+      <artifactId>akka-stream-alpakka-slick_$scalaBinaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.lightbend.akka", name: "akka-stream-alpakka-slick_$scalaBinaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
+
+You will also need to add the JDBC driver(s) for the specific relational database(s) to your project. Most of those database have drivers that are not available from public repositories so unfortunately some manual steps will probably be required. The Slick documentation has [information on where to download the drivers](http://slick.lightbend.com/doc/3.2.1/supported-databases.html).
+
+## Usage
+
+As always, before we get started we will need an @scaladoc[ActorSystem](akka.actor.ActorSystem) and a @scaladoc[Materializer](akka.stream.Materializer).
+
+Scala
+: @@snip (../../../../slick/src/test/scala/akka/stream/alpakka/slick/scaladsl/SlickSpec.scala) { #init-mat }
+
+Java
+: @@snip (../../../../slick/src/test/java/akka/stream/alpakka/slick/javadsl/SlickTest.java) { #init-mat }
+
+
+### Starting a Database Session
+
+All functionality provided by this connector requires the user to first create an instance of `SlickSession`, which is a thin wrapper around Slick's database connection management and database profile API.
+
+Scala
+: @@snip (../../../../slick/src/test/scala/akka/stream/alpakka/slick/scaladsl/SlickSpec.scala) { #init-session }
+
+Java
+: @@snip (../../../../slick/src/test/java/akka/stream/alpakka/slick/javadsl/SlickTest.java) { #init-session }
+
+As you can see, this requires you to configure your database using [typesafe-config](https://github.com/typesafehub/config) by adding a named configuration to your application.conf and then referring to that configuration when starting the session.
+
+Here is an example configuration for the H2 database, which is used for the unit tests of the Slick connector itself:
+
+Configuration
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-h2 }
+
+The configuration namespace is `akka.stream.alpakka.slick` and under that key you can specify multiple different database configurations, as long as yo use unique names. These can then be loaded by fully qualified name (so including the namespace prefix) using the `SlickSession.forConfig()` method described above.
```

Comment:
how is that config namespace used? is it used? why is it needed?

Reply:
exactly, now it looks like `akka.stream.alpakka.slick` is some magic prefix or something that we require

Reply:
Point taken. I'll simplify this to make sure it is not mistaken for a magic prefix.

Reply:
Fixed